### PR TITLE
Add fetch mailbox event

### PR DESCRIPTION
**Pull request by adamkempa**
First pass at this.

### CODE REVIEW

File: src/action-types.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
 export const FETCH_AUTH_TOKEN_REQUEST = 'FETCH_AUTH_TOKEN_REQUEST';
 export const FETCH_AUTH_TOKEN_DONE = 'FETCH_AUTH_TOKEN_DONE';
+export const FETCH_MAILBOX_DONE = 'FETCH_MAILBOX_DONE';
```
Comment (adamkempa) [inline src/action-types.ts]: - [ ] `FETCH_AUTH_MAILBOX_DONE`

File: src/reducer.ts [hunk 2]
```diff
@@ -1,24 +1,29 @@
 import {
```
Comment (adamkempa) [inline src/reducer.ts]: - [ ] undo all these changes in the reducer

File: src/reducers.spec.ts [hunk 3]
```diff
@@ -1,12 +1,13 @@
 import 'jest';
```
Comment (adamkempa) [inline src/reducers.spec.ts]: - [ ] undo these changes

File: src/utils.ts [hunk 4]
```diff
@@ -30,15 +31,22 @@ export interface TokenResponse {
   lift_session: {
     authenticated: Boolean;
     token: Token;
-  };
+  },
+  mailboxes: Array<Mailbox>; 
 }
 
 export const getAuthTokenFromResponse = (response: TokenResponse): Token => {
+  console.log(response);
```
Comment (adamkempa) [inline src/utils.ts]: - [ ] remove console.log

File: src/utils.ts [hunk 5]
```diff
@@ -30,15 +31,22 @@ export interface TokenResponse {
...
   return response.lift_session.authenticated
     ? response.lift_session.token
     : '';
 };
 
+export const getMailboxesFromResponse = (response: TokenResponse): Array<Mailbox> => {
+  console.log(response);
```
Comment (adamkempa) [inline src/utils.ts]: - [ ] remove console.log

**Comment by adamkempa** [comment]
@neurosnap I think this is good for review.

**Comment by adamkempa** [comment]
@neurosnap Should those CI tests be passing? (See below).

**Comment by neurosnap** [comment]
Yes they should.  run `npm test` locally

**Comment by adamkempa** [comment]
@neurosnap Tests are passing 💯
